core/bentley/src/core-bentley.ts

```diff
@@ -40,6 +41,10 @@ export * from "./UtilityTypes";
  * @docs-group-description BeSQLite
  * Classes for working with SQLite databases. SQLite underlies IModelDb and ECDb - see [Executing ECSQL]($docs/learning/ECSQL.md)
  */
+/**
+ * @docs-group-description ClassUtils
```

Comment:
No reason to create a new documentation group that only contains `@internal` functions.

Reply:
well I tried to confirm auto-merge, reloaded the page when it glitched out, and hit the same button which apparently was now "merge" instead of confirm auto merge. Please consider resolving this, or let me know if there are issues and I'll put up a new PR

Reply:
I did address this comment before merging